Makefile

```diff
@@ -30,95 +30,27 @@ help: ## Show this help message
 	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "  %-20s %s\n", $$1, $$2}' $(MAKEFILE_LIST)
 
 # Build targets for individual images
-.PHONY: build-dpdk-testpmd
-build-dpdk-testpmd: ## Build dpdk-testpmd image
-	podman build \
+.PHONY: build-%
+build-%: ## Build a specific image by name
+	@echo "Pushing $*..."
+	@podman build \
```

Comment:
Hmm... I like being able to see the progress of the "podman build" - will this remove that?